src/Magnum/Audio/PlayableGroup.h

```diff
@@ -0,0 +1,194 @@
+    friend Playable<dimensions>;
+
+    public:
+
+        /**
+         * @brief Constructor
+         */
+        explicit PlayableGroup(): SceneGraph::FeatureGroup<dimensions, Playable<dimensions>, Float>{} {}
+
+        /**
+         * @brief Play all sound sources in this group
+         * @return Reference to self (for method chaining)
+         */
+        PlayableGroup<dimensions>& playAll() {
+            for(UnsignedInt i = 0; i < this->size(); ++i)
+                (*this)[i].source().play();
+
+            return *this;
+        }
+
+        /**
+         * @brief Pause all sound sources in this group
+         * @return Reference to self (for method chaining)
+         */
+        PlayableGroup& pauseAll() {
+            for(UnsignedInt i = 0; i < this->size(); ++i)
+                (*this)[i].source().pause();
+
+            return *this;
+        }
+
+        /**
+         * @brief Stop all sound sources in this group
+         * @return Reference to self (for method chaining)
+         */
+        PlayableGroup& stopAll() {
+            for(UnsignedInt i = 0; i < this->size(); ++i)
+                (*this)[i].source().stop();
+
+            return *this;
+        }
+
+        /** @brief Gain */
+        Float gain() const {
+            return _gain;
+        }
+
+        /**
+         * @brief Set gain for all sound sources of Playables in this group
+         * @param gain Gain
+         *
+         * Will calculate the sound sources gain relative to the gain of the Playable
+         * and this playable group. The sources gain is computed as
+         * `sourceGain = playableGain*groupGain`.
+         * @return Reference to self (for method chaining)
+         */
+        PlayableGroup& setGain(const Float gain) {
+            _gain = gain;
+            for(UnsignedInt i = 0; i < this->size(); ++i)
+                (*this)[i].cleanGain();
+
+            return *this;
+        }
+
+        /**
+         * @brief Sound transformation
+         */
+        const Matrix3x3& soundTransformation() const {
+            return _soundTransform;
+        }
+
+        /**
+         * @brief Set transformation of the sounds in this group
+         * @return Reference to self (for method chaining)
+         */
+        PlayableGroup& setSoundTransformation(const Matrix3x3& matrix) {
+            _soundTransform = matrix;
+
+            for(UnsignedInt i = 0; i < this->size(); ++i)
+                (*this)[i].object().setDirty(); // TODO: Only if matrix is actually different?
+
+            return *this;
+        }
+
+        /**
+         * @brief Clean all the contained Playables.
+         *
+         * @see @ref AbstractObject::setClean()
+         */
+        void setAllClean() {
+            for(UnsignedInt i = 0; i < this->size(); ++i)
+                (*this)[i].object().setClean();
+        }
+
+    private:
+        Matrix3x3 _soundTransform;
+        Float _gain;
```

Comment:
I think it may be wise to initialize this to some value (`1.0f`?) (and then mentioning that in the `setGain()` docs). Otherwise there would be some random value which can kill your eardrum ;)